### ISSUE TITLE
Remove political propaganda please

### ISSUE THREAD

**Issue by Maxwelljones** [mezz/JustEnoughItems#2127]
Hi there,

I am not related to anything political in the USA. I live about 5000km away and I personally let you take care of your own governement. I have enough worries in my own country.

I have a problem though when this kind of political stuff appears on loading my modpack for everyone playing it, doesn't matter the location, which is in my eyes very wrong.

Please do remove this it's really annoying. Mods are videogames and should do politics. Please keep that for your twitter or whatever social account you have. Most of the planet don't care who you elect.

**Comment by FireController1847** [comment]
On one hand I agree, but on the other this is the authors opinion and he wants to share it with his fellow Americans, whom of which often need to be reminded. You get rid of the message once, and it doesn't bother you again. It's not like it shows up every time you open the same. Politics are part of life, stop trying to avoid them.

**Comment by Maxwelljones** [comment]
@FireController1847 I understand one would like to help people to grow, get involved in whatever cause he'd like to support, saving the dolphins, exploring space, helping people in need.. 
It's fine to do it in your own social media twitter, fb, ig, whatever. 

Here it's a minecraft mod, the propaganda shows up at every mod update, and whenever a modpack is updated if the mod-packer isnt clever enough to include the specific file in his pack.

I don't understand why someone would try to use a mod to push their own political opinions. Politics and videogames targeting **kids that are between 6 and 13** is a really bad mix. They can't vote and have no clue what it politics means.

**Comment by FireController1847** [comment]
@Maxwelljones My apologies. I read more into the incident and I retract my statement. Besides, all of this is for a mute point anyways. The author made a choice, he's not going to revert it.

This is a duplicate of #2093

**Comment by Maxwelljones** [comment]
@FireController1847 thanks for your understanding. Wish you all the best ;)

**Comment by ItzNxthaniel** [comment]
> I don't understand why someone would try to use a mod to push their own political opinions. Politics and videogames targeting kids that are between 6 and 13 is a really bad mix. They can't vote and have no clue what it politics means.

When I saw the vote message when I started up the game, I was fuming. Because I'm 16 and can't vote. As I said in a direct message to someone. "Yes saying to vote is one thing, using your modpack to say to vote for a specific person is another. But it's right and wrong at the same time. I don't need this bs in Minecraft." While I am very political, which one can vouch for me, I am sick and tired of this bs. Entering gaming, "YouTubers, Songs, and Shit is another thing."
